dcos/cluster.py

```diff
@@ -276,12 +276,20 @@ def get_cluster(name):
     """
     :param name: name of cluster
     :type name: str
-    :returns: Cluster identified by name
+    :returns: Cluster identified by name or ID, it also accepts an intial
+              portion of ID if it's unique.
```

Comment:
Is it the same in Docker? It sounds quite strange to match portion of IDs.

Reply:
it only work with the initial portion of ID, and if it's the unique match.